### ISSUE TITLE
#announce: was sent to nil

### ISSUE THREAD

**Issue by hernanmd** [tinchodias/FFICallLogger#2]
After installing FFICallLogger in a clean Pharo 11 

```smalltalk
EpMonitor disableDuring: [ 
	Metacello new
		baseline: 'FFICallLogger';
		repository: 'github://tinchodias/FFICallLogger';
		load ].
```
and trying to open the window with `TFLMainPresenter open` this error appears: #announce: was sent to nil

The problem seems to be the logger does `self reset.` before the announcer is set with `Announcer new`. See `MemoryLogger>>initialize`, so it could be a Beacon issue?

**Comment by tinchodias** [comment]
Thanks for the report and debugging. 

Did you try your idea about Beacon? It's possible to be related with Beacon since I normally load this project in conjunction with Bloc, which loads it from feenk I think.

Note: I started using Beacon to log but at some point realized that the benefits over using directly Announcements were not clear. I remember I checked that avoiding Beacon didn't imply big changes.

**Comment by hernanmd** [comment]
Hi Tincho. I understand what you're saying but I have no idea what could be the happy path.
If it were for me I would make Beacon optional, and provide raw Transcript show, or better, Stdio show.

I just hardcoded an "announcer := Announcer new" in one of the Beacon's initialize methods. So "make it pass".

I could give a look once I return from vacations :)

**Comment by tinchodias** [comment]
Dependency browser shows:
<img width="339" alt="Screen Shot 2022-10-03 at 18 52 51" src="https://user-images.githubusercontent.com/3044265/193692649-58e3eb65-0911-46dc-830a-f97a71d5254f.png">

**Comment by tinchodias** [comment]
Hernan! when I debugged it far more simple than I expected to it is fixed in 8bf2fea4821409371b2d880d3e629394bd784863

**Comment by tinchodias** [comment]
Hope you can use it now. Any feedback is welcome, thanks!
